server/src/test/java/org/elasticsearch/script/StatsSummaryTests.java

```diff
@@ -71,13 +71,13 @@ public void testEqualsAndHashCode() {
         assertThat(stats1.hashCode(), equalTo(stats2.hashCode()));
 
         // Accumulators with same values are equals
-        randomDoubles(randomIntBetween(0, 20)).forEach(stats1.andThen(stats2));
+        randomDoubles(randomIntBetween(1, 1000)).forEach(stats1.andThen(stats2));
         assertThat(stats1, equalTo(stats2));
         assertThat(stats1.hashCode(), equalTo(stats2.hashCode()));
 
         // Accumulators with different values are not equals
-        randomDoubles(randomIntBetween(0, 20)).forEach(stats1);
-        randomDoubles(randomIntBetween(0, 20)).forEach(stats2);
+        randomDoubles(randomIntBetween(1, 1000)).forEach(stats1);
+        randomDoubles(randomIntBetween(1, 1000)).forEach(stats2);
```

Comment:
Would it make sense to even keep the random ranges non-overlapping in this case? Or is this overkill because picking the same amount of doubles _and_ the same values is so unlikely?

Reply:
My assumption was indeed that the probability of having an equality with such a combination is lower than winning to the lottery twice the same month ....

Also, I did run the tests with `-D tests.iters=1000000` to check it